src/scripts/main.js

```diff
@@ -1,3 +1,368 @@
+        if (columnName === 'Name') {
+          sortDESC(columnName, 0);
+        }
+
+        if (columnName === 'Position') {
+          sortDESC(columnName, 1);
+        }
+
+        if (columnName === 'Office') {
+          sortDESC(columnName, 2);
+        }
+
+        if (columnName === 'Age') {
+          sortDESC(columnName, 3);
+        }
+
+        if (columnName === 'Salary') {
+          sortDESC(columnName, 4);
+        }
+      }
+    }
+
+    getTHead.forEach((item) => {
+      item.addEventListener('click', function () {
+        getTBody.forEach((del2) => del2.classList.remove('active'));
+        getTHead.forEach((del) => del.classList.remove('active'));
+        getTFooter.forEach((itemf) => itemf.classList.remove('active'));
+
+        item.classList.add('active');
+      });
+    });
+  });
+});
+
+// Додаємо обробник подій для рядків таблиці
+getTBody.forEach((item2) => {
+  item2.addEventListener('click', function () {
+    getTHead.forEach((del2) => del2.classList.remove('active'));
+    getTBody.forEach((del2) => del2.classList.remove('active'));
+    getTFooter.forEach((itemTfooter) => itemTfooter.classList.remove('active'));
+
+    item2.classList.add('active');
+  });
+});
+
+const createElementForm = document.createElement('form');
+
+createElementForm.classList.add('new-employee-form');
+
+const textContentLabel = [
+  'Name: ',
+  'Position: ',
+  'Office: ',
+  'Age: ',
+  'Salary: ',
+];
+
+// звертатись по масиву
+const arrForLabel = [];
+
+textContentLabel.forEach((item) => {
+  const createElementLabel = document.createElement('label');
+  const createElementInput = document.createElement('input');
+  const cleanText = item.replace(': ', '').toLowerCase();
+
+  createElementLabel.textContent = cleanText;
+  createElementLabel.textContent = item;
+  arrForLabel.push(createElementLabel);
+  createElementInput.setAttribute('name', cleanText.toLowerCase());
+  createElementInput.setAttribute('data-qa', cleanText.toLowerCase());
+  createElementForm.append(createElementLabel);
+
+  if (
+    createElementInput.name === 'age' ||
+    createElementInput.name === 'salary'
+  ) {
+    createElementInput.type = 'number';
+  }
+
+  if (
+    createElementInput.name === 'name' ||
+    createElementInput.name === 'position'
+  ) {
+    createElementInput.type = 'text';
+  }
+
+  if (createElementInput.name === 'office') {
+    const select = document.createElement('select');
+
+    select.setAttribute('name', cleanText.toLowerCase());
+
+    const cities = [
+      'Tokyo',
+      'Singapore',
+      'London',
+      'New York',
+      'San Francisco',
+    ];
+
+    cities.forEach((city) => {
+      const option = document.createElement('option');
+
+      option.value = city;
+      option.textContent = city;
+      select.append(option);
+    });
+    createElementLabel.append(select);
+  }
+
+  if (createElementInput.name !== 'office') {
+    createElementLabel.append(createElementInput);
+  }
+});
```

Comment:
The code for creating form labels and inputs is repetitive. Consider refactoring this section to reduce redundancy and improve maintainability. You could create a helper function to handle the creation and configuration of form elements.